docs/legacy/copied-from-beats/docs/command-reference.asciidoc

```diff
@@ -432,65 +430,6 @@ Specifies the name of the command to show help for.
 {beatname_lc} help export
 -----
 
-ifndef::serverless[]
-[float]
-[[keystore-command]]
-==== `keystore` command
-
-{keystore-command-short-desc}.
-
-*SYNOPSIS*
-
-["source","sh",subs="attributes"]
-----
-{beatname_lc} keystore SUBCOMMAND [FLAGS]
-----
-
-*`SUBCOMMAND`*
-
-*`add KEY`*::
-Adds the specified key to the keystore. Use the `--force` flag to overwrite an
-existing key. Use the `--stdin` flag to pass the value through `stdin`.
-
-*`create`*::
-Creates a keystore to hold secrets. Use the `--force` flag to overwrite the
-existing keystore.
-
-*`list`*::
-Lists the keys in the keystore.
-
-*`remove KEY`*::
-Removes the specified key from the keystore.
-
-*FLAGS*
-
-*`--force`*::
-Valid with the `add` and `create` subcommands. When used with `add`, overwrites
-the specified key. When used with `create`, overwrites the keystore.
-
-*`--stdin`*::
-When used with `add`, uses the stdin as the source of the key's value.
-
-*`-h, --help`*::
-Shows help for the `keystore` command.
-
-
-{global-flags}
-
-*EXAMPLES*
-
-["source","sh",subs="attributes"]
------
-{beatname_lc} keystore create
-{beatname_lc} keystore add ES_PWD
-{beatname_lc} keystore remove ES_PWD
-{beatname_lc} keystore list
------
-
-See <<keystore>> for more examples.
-
-endif::[]
-
```

Comment:
Not blocking this PR, but let's have a conversation around how to document the commands for the apm-server binary before releasing the new docs (including why/if to remove the keystore docs);